go/engine/bootstrap.go

```diff
@@ -85,12 +88,14 @@ func (e *Bootstrap) Run(ctx *Context) error {
 	if e.G().ConnectivityMonitor.IsConnected(context.Background()) == libkb.ConnectivityMonitorYes {
 		e.G().Log.Debug("connected, running full tracker2 syncer")
 		if err := libkb.RunSyncer(ts, e.status.Uid, false, nil); err != nil {
-			return err
+			e.G().Log.Warning("error running Tracker2Syncer: %s", err)
+			return nil
```

Comment:
Not sure if @malgorithms cares, but I guess the color coding change might be confusing? I'd be curious if people pick up on the blue/green state in chat anyway though, seems extremely subtle. There is no obvious "we don't know" state those names could be in, I agree with you it seems better that the app should start, as opposed to not starting because we can't color the names correctly.